apostrophecms/apostrophe
# add metaType property to newly inserted widgets

## Summary

Add `metaType` to newly inserted widgets

## What are the specific steps to test this change?

1. Add a widget to an area
2. With the `vue-dev-tools`, check that the `metaType` property is set after insert
3. Check that the `metaType` property is still there after a widget update
4. Check that the `metaType` property is still there after a full page reload

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
